src/client/AM.PA.MonitoringTool/SlackTracker/Views/FirstStartWindow.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<UserControl x:Class="SlackTracker.Views.FirstStartWindow"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:SlackTracker.Views"
+             xmlns:paColors="clr-namespace:Shared;assembly=Shared"
+             mc:Ignorable="d" 
+             d:DesignHeight="300" d:DesignWidth="300">
+    <Grid Background= "White">
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        
+        <StackPanel Margin="10" Grid.Row="0">
+            <TextBlock TextWrapping="Wrap">
+                The Slack Tracker is used to collect your activity on different slack channels with your private access token.
+                <LineBreak  /><LineBreak  />
+                <CheckBox Name="Enabled" Checked="CheckBox_Checked" Unchecked="CheckBox_Unchecked">Enable Slack Tracker </CheckBox>
+                <LineBreak ></LineBreak><LineBreak></LineBreak>
+            </TextBlock>
+
+            <TextBlock Name="ThanksMessage" TextWrapping="Wrap" Visibility="Collapsed" Foreground="{x:Static paColors:Settings.RetrospectionColorBrush}">
+                Thanks for enabling the Slack Tracker. When Clicking "Next", You will need to add the channels and their token for which you want your activity to be tracked.
```

Comment:
Maybe rewrite to something like:

"Thank you for enabling the Slack Tracker. After you click "Next", you will be asked to log-into your Slack-account and select the channels that are considered in the analysis."


btw, can the API tell you which channels a user registered to? i.e., can we have a default that takes all channels the user joined?